helper/resource/testing.go

```diff
@@ -700,6 +700,23 @@ type TestStep struct {
 	// [plancheck]: https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/plancheck
 	ImportPlanChecks ImportPlanChecks
 
+	// ImportPlanVerify checks that a generated plan for a plannable import
+	// (ImportBlockWithID or ImportBlockWithResourceIdentity) agrees with
+	// the known state of a previous test stepstate values
+	// that are finally put into the state after import match for all the
+	// IDs returned by the Import.  Note that this checks for strict equality
+	// and does not respect DiffSuppressFunc or CustomizeDiff.
+	//
+	// By default, the prior resource state and import resource state are
+	// matched by the "id" attribute. If the "id" attribute is not implemented
+	// or another attribute more uniquely identifies the resource, set the
+	// ImportStateVerifyIdentifierAttribute field to adjust the attribute for
```

Comment:
The PR description states that `ImportStateVerifyIdentifierAttribute` is not mirrored – am I missing something or would this comment need an update?

Reply:
The comment needs an update. Nice catch 😃 I've pushed a commit to change this.